app/helpers/albums_helper.rb

```diff
@@ -0,0 +1,2 @@
+module AlbumsHelper
+end
```

Comment:
Depending on who you ask, some people feel strongly about deleting the auto-generated files that you don't use.

It's not a big deal, though, and maybe not worth your time.